### DISCUSSION TITLE
Ability to change icon background of value metric

### DISCUSSION THREAD

**Discussion by brain82** [laravel/nova-issues#5266] [Ideas & Feature Requests]
Hi

It will be nice to be able to change the background color of the value metric icon.

One of my use cases is displaying the count of overdue tasks. If there's an overdue task, I use an exclamation icon otherwise the tick icon (see screenshots below). It would be nice if I could change the background color to yellow when there are overdue tasks.

<img width="578" alt="image" src="https://user-images.githubusercontent.com/6013683/213977154-f8609242-f8de-49a3-a1fd-ec62aae69510.png">

<img width="579" alt="image" src="https://user-images.githubusercontent.com/6013683/213977189-7b947020-92cb-4bdf-bef7-d5179d6297ae.png">

**Comment by bkuhl** [comment]
duplicate of https://github.com/laravel/nova-issues/discussions/5239
